packages/manager/modules/iam/src/components/policyIdentities/policyIdentities.controller.js

```diff
@@ -0,0 +1,106 @@
+import { ROUTES } from '@iam/routes';
+import { decodeUrn, encodeUrn } from '@iam/resolves';
+import { URN_VERSION, ENTITY } from '@iam/constants';
+
+export default class PolicyIdentitiesController {
+  /* @ngInject */
+  constructor($q, $translate, IdentityService, PolicyService, coreConfig) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.IdentityService = IdentityService;
+    this.PolicyService = PolicyService;
+    this.region = coreConfig.getRegion();
+    this.user = coreConfig.getUser();
+  }
+
+  $onInit() {
+    this.loading = true;
+
+    this.identities = this.policy.identities
+      .map(decodeUrn)
+      .map((urn) => ({ title: urn.components.slice(-1).toString(), urn }));
+
+    this.$q
+      .all({
+        users: this.IdentityService.getUsers(),
+        groups: this.IdentityService.getGroups(),
+      })
+      .then(({ users, groups }) => {
+        this.userList = [
+          ...users.map((user) => ({ accountType: 'user', userName: user })),
+          ...groups.map((group) => ({
+            accountType: 'group',
+            userName: group,
+          })),
+        ];
+      })
+      .catch(() => {
+        this.userList = [];
```

Comment:
By making the API error silent no one is able to report it
Please use the classic message mechanism (`alertResolve` is meant for that)

Reply:
It's applied